Rakefile

```diff
@@ -20,7 +20,7 @@ desc("Run unit tests")
 Rake::TestTask.new(:unit_test) do |t|
   t.libs << "test"
   t.libs << "lib"
-  t.test_files = FileList['test/unit/**/*_test.rb']
+  t.test_files = FileList['test/unit/**/*_test.rb', 'test/exe/**/*_test.rb']
```

Comment:
Is this a unit test or should it be a new category?

Reply:
`test/cli`, perhaps?